app/views/admin/enterprise_fees/_calculator_settings.html.haml

```diff
@@ -7,6 +7,9 @@
       - if !enterprise_fee.new_record?
         .calculator-settings
           = f.fields_for :calculator do |calculator_form|
-            = preference_fields(enterprise_fee.calculator, calculator_form)
+            - fields = preference_fields(enterprise_fee.calculator, calculator_form)
```

Comment:
I'd make these lines a one liner, and never use a single letter variable:
`= preference_fields(enterprise_fee.calculator, calculator_form).each do |field|`

Reply:
👍 https://github.com/openfoodfoundation/openfoodnetwork/pull/5998/commits/96fb65600e918a56c22c944d533f8d116106a88e